### PR TITLE
Update the CLI command between 2 readme to look consistent

### PR DESCRIPTION
**Pull request by haishengwu-okta**

### CODE REVIEW

File: okta-hosted-login/README.md [hunk 1]
```diff
@@ -32,11 +32,16 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret} \
-    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) configure this property to enable SSO logout. 
+    -Dokta.oauth2.clientSecret={clientSecret}
 ```
 
-> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead update the projects `application.yml`
+> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead update the projects `application.yml` if you're using a closed source control system. Otherwise, we recommend you store them as environment variables. For example:
```
Comment (bdemers) [inline okta-hosted-login/README.md:38]: I wouldn't add the "closed source" bit. Those secrets still _shouldn't_ be checked in, they should be managed by some automation process when used in a file format (but that's harder to explain, so maybe just push the ENV var route)

```suggestion
> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead, we recommend you store them as environment variables. For example:
```

File: okta-hosted-login/README.md [hunk 2]
```diff
@@ -32,11 +32,16 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret} \
-    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) configure this property to enable SSO logout.
```
Comment (arvindkrishnakumar-okta) [inline okta-hosted-login/README.md:36]: Why is this `postLogoutRedirectUri` property removed? I know its optional but still good to note it I think.

Comment (haishengwu-okta) [reply okta-hosted-login/README.md:36]: I saw it's hard coded in the code plus it is not mentioned in the other readme. The readme at our java spring sdk does mention this so I thought we can simplify the cli here in sample. If we need to know detail, the instinct would go to underlining SDK page.

```
// set logout URL
.and().logout().logoutSuccessUrl("/")
```

File: okta-hosted-login/README.md (hunk 1 above)
Comment (haishengwu-okta) [reply okta-hosted-login/README.md:38]: updated both readme.

**Review by arvindkrishnakumar-okta — APPROVED** [review]
lgtm

**Pull request merged**